docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+
+### Signature Type (typ)
+
+We use the `typ` value in the protected header of each signature to distinguish the "type" of signature it is. This is already an established pattern with OpenID Provider signatures in ID Tokens having `typ=JWT`.
+As shown, we have three signatures types:
+
+1. `typ=JWT` **OP (OpenID Provider) signature and protected header:** The first signature is the signature of the party that issued the ID Token, that is, the signature of the OpenID provider.
+2. `typ=CIC` **CIC (Client-Instance Claims) signature and protected header:** The second signature is generated by the identity's client. This signature contains user's public key.
+3. `typ=COS` **COS (Cosigner) signature and protected header:** The third signature is the COS (Cosigner) signature. The Cosigner is a third party who has independently authenticated the identity. It exists to remove the OpenID Provider as a single point of compromise. The COS signature is optional and not every PK Token will have one. It is up to OpenPubkey verifiers to decide if they require a Cosigner signature or not.
+
+### Commitment mechanism
+
+**CIC Always Committed to by OP's Signature:**
+The CIC protected header contains the identity's public key and associated metadata. To bind the identity's public key to the ID Token, the CIC is always committed to in the payload or the OP's protected header. Why payload or OP's protected header? Because the OP's signature signs both the payload and the protected header. This means the binding between identity and public key is made by the party that authenticated the identity and determined the identity is are who they say they are.
+
+In our PK Token zoo below we describe four different mechanisms that the CIC can be committed to under the OP's signature:
+
+| Commitment Type               | Commitment Location  | Claim name |
+| -------------                 |--------------        |--------------|
+| Nonce-Commitment              | `payload`            | `nonce`      |
+| Audience-Commitment           | `payload`            | `aud`        |
+| GQ-Commitment                 | `OP-protected header`| `CIC`        |
+| ZK-Commitment                 | `OP-protected header`| `CIC`        |
+
+### Types of Signatures in a PK Token
+
+#### OP (OpenID Provider) Signature
+
+This is signature of the OpenID Provider that created the payload and signed it to create the ID Token. This signature is responsible for binding the identity claims in the payload to that identity's public key.
+
+Typically OP signature/protected header along with the payload is the ID Token. For some more advanced forms of PK Tokens such as GQ or ZK PK Tokens, we transform OP signature and protected header. In these cases, you can no longer recover the ID Token from the PK Token, but even when we alter the OP Signature and protected header in this way, we don't break the cryptographic binding between payload and OpenID Provider that the signature provides.
```

Comment:
Similar feedback - sometimes we have OP signature and others OP Signature. Suggest sticking to one format for consistency

Reply:
I think I got all of them